l10n_it_pec/view/partner_view.xml

```diff
@@ -13,5 +13,30 @@
             </field>
         </record>
 
+        <record id="view_partner_pec_tree" model="ir.ui.view">
+            <field name="name">res.partner.pec.tree</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="base.view_partner_tree"/>
+            <field name="arch" type="xml">
+                <field name="email" position="after">
+                    <field name="pec_mail" />
+                </field>
+            </field>
+        </record>
+
+        <record model="ir.ui.view" id="res_partner_kanban_pec_view">
+            <field name="name">res.partner.pec.kanban</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="base.res_partner_kanban_view"/>
+            <field name="arch" type="xml">
+                <field name="email" position="after">
+                 <field name="pec_mail"/>
+                </field>
+                  <xpath expr="//div[contains(@class, 'oe_kanban_details')]/ul" position="inside">
+                                <li t-if="record.pec_mail.raw_value"><b>PEC: </b><a t-attf-href="mailto:#{record.pec_mail.raw_value}"><field name="pec_mail"/></a></li>
```

Comment:
Please fix indentation and use "address@email.com (PEC)" instead of "**PEC:** address@email.com" (same way as the 8.0 branch)